packages/ag-charts-community/src/chart/interaction/tooltipManager.ts

```diff
@@ -67,12 +67,13 @@ export class TooltipManager {
         }
 
         const canvasRect = this.domManager.getBoundingClientRect();
+        const boundingRect = this.tooltip.bounds === 'extended' ? this.domManager.getOverlayClientRect() : canvasRect;
```

Comment:
Minor - but maybe this could be `auto` - as in automatically determine the correct element. Potentially you'd later add support for specifying an explicit HTMLElement